docs/src/RunningOnGPUs.md

```diff
@@ -0,0 +1,45 @@
+# Running on GPUs
+
+ITensor provides package extensions for running tensor operations in the GPU with different backends.
+You can activate a backend by loading the appropriate Julia GPU package alongside ITensors.jl
+and moving your tensors to the GPU using that package's provided accelerator.
+
+For example, you can load CUDA.jl to perform tensor operations on NVIDIA GPUs or Metal.jl to perform tensor operations on Apple GPUs:
+
+```julia
+using ITensors
+
+i, j, k = Index.((2, 2, 2))
+A = randomITensor(i, j)
+B = randomITensor(j, k)
+
+# Perform tensor operations on CPU
+A * B
+
+###########################################
+using CUDA # This will trigger the loading of `NDTensorsCUDAExt` in the background
+
+# Move tensors to NVIDIA GPU
+Agpu = cu(A)
+Bgpu = cu(B)
+
+# Perform tensor operations on NVIDIA GPU
+Agpu * Bgpu
```

Comment:
```suggestion
Acu = cu(A)
Bcu = cu(B)

# Perform tensor operations on NVIDIA GPU
Acu * Bcu
```

Reply:
Also `cu` provided by `CUDA.jl` will convert `Float64` to `Float32`,...  Using `NDTensors.cu` preserves the element type.

Reply:
@kmp5VT I would rather keep `NDTensors.cu` hidden for now and just encourage users to use `CUDA.cu`.

Reply:
Do you think there should be a warning about this conversion in the docs or is this CUDA.jl's responsibility? I, for one, had no idea this happens.

Reply:
I think it is up to users/CUDA.jl's responsibility to document this. We can add something like "Please see the documentation of the corresponding GPU packages for more details on conversion functions like `cu` and `mtl`." I don't think we should be responsible for re-documenting the behavior of those packages.